CPT4/vocab_specific_QA.sql

```diff
@@ -0,0 +1,30 @@
+-- Our aim is to build a unique hierarchy of Procedures with CPT4 embedded in SNOMED/OMOP Ext hierarchy.
+--- The script below retrieves the number of concepts in hierarchy against the number of concepts that are not yet in hierarchy.
+--- Use this counts for analysis and renew respective numbers in https://github.com/OHDSI/Vocabulary-v5.0/wiki/Known-Issues-in-Vocabularies
+
+WITH concepts_in_hierarchy AS (SELECT DISTINCT cr.concept_id_1
+                                FROM concept_relationship cr
+                                        JOIN concept c ON cr.concept_id_1 = c.concept_id
+                                        JOIN concept c1 ON cr.concept_id_2 = c1.concept_id
+                               WHERE c.vocabulary_id = 'CPT4'
+                                 AND c.concept_class_id = 'CPT4'
+                                 AND c1.vocabulary_id IN ('SNOMED', 'OMOP Extension')
+                                 AND cr.relationship_id IN ('Is a', 'Maps to')
```

Comment:
Do we call it a hierarchy? "Is a" combined with "Maps to" may potentially create ambiguity. I suggest adding 
..retrieves the number of concepts in hierarchy through either Is a or Maps to relationships...

Reply:
Fixed

Reply:
@Alexdavv disagreed in email

Reply:
Changed the QA logic a bit. Have a look, please.